netci.groovy

```diff
@@ -0,0 +1,144 @@
+import jobs.generation.Utilities;
+
+def project = GithubProject
+def branch = GithubBranchName
+
+static void addMultiScm(def myJob, boolean isPR) {
+  if (isPR) {
+    addPRTestMultiScm(myJob)
+  }
+  else {
+    addMultiScm(myJob)
+  }
+}
+
+static void addPRTestMultiScm(def job) {
+  job.with {
+    multiscm {
+      git {
+        remote {
+          github('Microsoft/llvm')
+        }
+        branch('*/MS')
+        wipeOutWorkspace(true)
+        shallowClone(true)
+        relativeTargetDir('LLVM')
+      }
+      git {
+        remote {
+          github('dotnet/coreclr')
+        }
+        branch('*/master')
+        relativeTargetDir('coreclr')
+      }
+      git {
+        remote {
+          github('dotnet/llilc')
+          refspec('${GitRefSpec}')
+          url('${GitRepoUrl}')
+        }
+        branch('${sha1}')
+        relativeTargetDir('LLVM\\tools\\llilc')
+      }
+    }
+  }
+}
+
+static void addMultiScm(def myJob) {
+  myJob.with {
+    multiscm {
+      git {
+        remote {
+          github('Microsoft/llvm')
+        }
+        branch('*/MS')
+        wipeOutWorkspace(true)
+        shallowClone(true)
+        relativeTargetDir('LLVM')
+      }
+      git {
+        remote {
+          github('dotnet/llilc')
+        }
+        branch('*/master')
+        relativeTargetDir('LLVM\\tools\\llilc')
+      }
+      git {
+        remote {
+          github('dotnet/coreclr')
+        }
+        branch('*/master')
+        relativeTargetDir('coreclr')
+      }
+    }
+  }
+}
+
+[true, false].each { isPR ->
+  ['Debug', 'Release'].each { configuration ->
+    def newJobName = Utilities.getFullJobName(project, configuration, isPR)
```

Comment:
Change the second parameter if you want to add more info to change the generated job name:

e.g.

"windows_${configuration}"